Tests/tests_e2e/content/xsoar_saas/test_e2e_xsoar_saas.py

```diff
@@ -54,21 +54,28 @@ def test_edl_returns_indicators(request: SubRequest, xsoar_saas_client: XsoarSaa
         username = integration_params["credentials"]["identifier"]
         password = integration_params["credentials"]["password"]
 
-        with save_integration_instance(
-            xsoar_saas_client,
-            integration_params=integration_params,
-            integration_id="EDL",
-            is_long_running=True,
-            instance_name=get_integration_instance_name(integration_params, default="EDL")
-        ) as edl_instance_response:
-            instance_name = edl_instance_response.get("name")
-
-            edl_response = xsoar_saas_client.do_long_running_instance_request(
-                instance_name, username=username, password=password
-            )
-            assert edl_response.text, f'could not get indicators from url={edl_response.request.url} from ' \
-                f'instance {instance_name} with available indicators={available_indicators},' \
-                f' status code={edl_response.status_code}, response={edl_response.text}'
+        for i in range(1, 6):
```

Comment:
what happens if we have more than 6 failed attempts ? 
Do you fail the test ?

Reply:
you are right, fixed